### PR TITLE
Add more set:sys Get*/Set* functions

### PR DESCRIPTION
**Pull request by joel16**
These are just some simple IPCs really.

Added the following:

- setsysSetLockScreenFlag
- setsysSetConsoleInformationUploadFlag
- setsysSetAutomaticApplicationDownloadFlag
- setsysGetQuestFlag
- setsysSetQuestFlag
- setsysGetUsb30EnableFlag
- setsysSetUsb30EnableFlag
- setsysSetNfcEnableFlag
- setsysSetWirelessLanEnableFlag
- setsysSetBluetoothEnableFlag
- setsysGetAutoUpdateEnableFlag
- setsysSetAutoUpdateEnableFlag
- setsysGetBatteryPercentageFlag
- setsysSetBatteryPercentageFlag
- setsysGetExternalRtcResetFlag
- setsysSetExternalRtcResetFlag
- setsysGetUsbFullKeyEnableFlag
- setsysSetUsbFullKeyEnableFlag
- setsysGetBluetoothAfhEnableFlag
- setsysSetBluetoothAfhEnableFlag
- setsysGetBluetoothBoostEnableFlag
- setsysSetBluetoothBoostEnableFlag
- setsysGetInRepairProcessEnableFlag
- setsysSetInRepairProcessEnableFlag
- setsysGetHeadphoneVolumeUpdateFlag
- setsysSetHeadphoneVolumeUpdateFlag

### CODE REVIEW

**Comment by yellows8** [comment]
Can you impl an internal setsys get/set ipc func, which the get/set funcs call with the required cmdid?(These current funcs are mostly duplicates)

**Comment by joel16** [comment]
@yellows8 Yeah I thought about that but wasn't sure if you guys would accept it. I guess I should have asked first :p Anyways, would 2 functions (one for get, the other for set) and 2 enums (one for get, other for set)? work out fine?

**Comment by yellows8** [comment]
"enums" what

**Comment by joel16** [comment]
Enumerators??? 
Similar to how SPL uses it: https://github.com/switchbrew/libnx/blob/master/nx/include/switch/services/spl.h#L12

**Comment by yellows8** [comment]
What for though?(Not needed for cmdids)

**Comment by WinterMute** [comment]
This seems overly complex & bloated to me. 

Is there some reason this couldn't/shouldn't be done with two functions & an enum?

**Comment by joel16** [comment]
That's literally what I was asking.

**Comment by yellows8** [comment]
Guess that's fine since there's so many cmds.

**Comment by WinterMute** [comment]
LGTM

File: nx/include/switch/services/set.h [hunk 1]
```diff
@@ -33,6 +33,26 @@ typedef enum
     SetLanguage_Total,      ///< Total languages supported by this enum.
 } SetLanguage;
 
+/// Command IDS for setsysGetFlag/setsysSetFlag.
```
Comment (yellows8) [inline nx/include/switch/services/set.h]: IDs

File: nx/include/switch/services/set.h [hunk 2]
```diff
@@ -74,37 +94,15 @@ Result setsysGetColorSetId(ColorSetId* out);
...
- * @param out Pointer to output the status to. 
- */
-Result setsysGetNfcEnableFlag(bool *out);
-
-/**
- * @brief Gets the wireless LAN status.
- * @param out Pointer to output the status to. 
+ * @brief Gets the status of the specified settings flag
+ * @param flag The specified settings flag.
+ * @param out Pointer To get the status from.
```
Comment (yellows8) [inline nx/include/switch/services/set.h]: *to

File: nx/include/switch/services/set.h [hunk 3]
```diff
@@ -74,37 +94,15 @@ Result setsysGetColorSetId(ColorSetId* out);
...
- * @param out Pointer to output the status to. 
- */
-Result setsysGetNfcEnableFlag(bool *out);
-
-/**
- * @brief Gets the wireless LAN status.
- * @param out Pointer to output the status to. 
+ * @brief Gets the status of the specified settings flag.
+ * @param flag The specified settings flag.
+ * @param out Pointer to get the status from.
```
Comment (yellows8) [inline nx/include/switch/services/set.h]: "Output pointer for the status."

**Pull request merged**